packages/core/src/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const InjectionSymbols = {
-  MessagePickupRepository: Symbol('MessagePickupRepository'),
+  MessagePickupRepository: Symbol('MessagePickupRepository'), // TODO: Move to DIDComm
```

Comment:
I think we should get rid of the injectionsymbols, and move it to the corresponding packages/files.

In most cases it's not needed to use the specific injection token publicy (as you can get it from the agent config / agent context directly)

Reply:
I agree.. InjectionSymbols are somewhat a pattern that we used at the beginning of Credo development but now they can be replaced (at least most of them).

In this particular case probably we can work with events, that will be catched by the Message Pickup module in order to decouple both. Surely I'll think about it in a further PR in the incoming days.